mlbaklava/resources/process/Dockerfile

```diff
@@ -7,7 +7,7 @@
 FROM python:$python_version-slim
```

Comment:
We would also need parametrize this as there can be scenario where organization has blocked docker hub.

We are planning to parametrize this so that baklava can be adopted at Intuit.